pkg/primitives/vm/utils.go

```diff
@@ -25,6 +26,145 @@ var (
 	}
 )
 
+// fill up the VM (machine) object with write boot config for a full virtual machine (with a disk image)
+func (p *Manager) prepVirtualMachine(
+	ctx context.Context,
+	cloudImage string,
+	imageInfo FListInfo,
+	machine *pkg.VM,
+	config *zos.ZMachine,
+	deployment *gridtypes.Deployment,
+	wl *gridtypes.WorkloadWithID,
+) error {
+	var (
```

Comment:
Wondering why var block is needed here?

Reply:
it had more stuff then removed then the block remained